modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -366,7 +366,7 @@ FeatureFloodCount::execute()
       if (elem->processor_id() == rank)
       {
         if (hasBoundary(boundary_id))
-          for (MooseIndex(_vars) var_num = 0; var_num < _vars.size(); ++var_num)
+          for (const auto var_num : make_range(_vars.size()))
```

Comment:
```suggestion
          for (const auto var_num : index_range(_vars))
```

for future reference

Reply:
A damn... I did this enough that it's probably worth fixing. I'll push those changes and also remove an unused function that I orphaned and then we can wrap this up. Thanks!